integration/system/event_test.go

```diff
@@ -58,3 +66,63 @@ func TestEvents(t *testing.T) {
 	}
 
 }
+
+func TestEventsEmptyOutput(t *testing.T) {
```

Comment:
This test doesn't seem very valuable. I think we can just test the response code is 200 OK in the other test case.

Reply:
Thanks @dnephin, I have moved this test case into the other test and checks to make sure the response code is 200 and the response will return (non-blocking).